### DISCUSSION TITLE
what a brilliant effort 0.14 is

### DISCUSSION THREAD

**Discussion by ElectronicBattle** [blakeblackshear/frigate#12770] [General]
Hello all

Thanks to very supportive help, especially from from @NickM-27 and @hawkeye217 ,  I now have a second instance of Frigate running. This one, via docker-compose on my main machine, is 0.14 and it gives me access to all of the hardware I need. The ability to playback the in-browser video thumbnails is working now!

I am still using 0.13 as my main Frigate tool, probably until the update for trueNAS is available, but I might stick with the docker-compose 0.14 solution on my main PC anyway, due to the availability of a Radeon RX580 which can handle the various transcoding tasks.

The new UI is indeed excellent and I am slowly getting to grips with various confusions of my own to do with Frigate configuration. The new UI makes it easier; I am also new to docker so that's taken a while to get going today but I'll stick with it.

Thanks!

EB

**Comment by DJOma** [comment]
Indeed, this is a huge step forward.
I love the ability to see what happened at a quick glance und the configuration and motion tuning in live view is really nice.
